rest-service/manager_rest/rest/rest_utils.py

```diff
@@ -861,18 +862,50 @@ def get_labels_list(raw_labels_list):
         [(key, raw_value)] = label.items()
         values_list = raw_value if isinstance(raw_value, list) else [raw_value]
         for value in values_list:
-            if ((not isinstance(key, text_type)) or
-                    (not isinstance(value, text_type))):
-                _raise_bad_labels_list()
-            validate_inputs({'key': key, 'value': value})
-            if key.startswith(CFY_LABELS_PREFIX) and key not in CFY_LABELS:
```

Comment:
aha, so there was a bug before, this was checked before the `.lower()`

Reply:
You're absolutely right, I realized it while working on this task. You could put a `csys-` label id you used uppercase letters, but now you can't :)